packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -16,6 +18,10 @@ import { obfuscateNumber } from '../sms/base'
 
 const ATTESTATION_ERROR = 'Valid attestation could not be provided'
 const NO_INCOMPLETE_ATTESTATION_FOUND_ERROR = 'No incomplete attestation found'
+export const INVALID_SIGNATURE_ERROR = 'Signature is invalid'
+
+const odisPubKey = OdisUtils.Query.ODIS_MAINNET_CONTEXT.odisPubKey
```

Comment:
(apologies for commenting this in two places, old thread was outdated)
hey one thing on this @isabellewei, we should still make the odisPubKey configurable by network (i.e. perhaps via an optional env var that specifies the network and then use that to pick the proper ODIS context), otherwise I guess the logic will fail on alfajores/baklava?

Reply:
+1 @eelanagaraj how about something like this? 

```suggestion
let odisPubKey = OdisUtils.Query.ODIS_MAINNET_CONTEXT.odisPubKey
if(getCeloProviders().includes('https://alfajores-forno.celo-testnet.org') || 
   getCeloProviders().includes('https://baklava-forno.celo-testnet.org')) { 
     odisPubKey = OdisUtils.Query.ODIS_ALFAJORES_CONTEXT.odisPubKey 
 }
```

Note: there is no ODIS baklava context 


>

Reply:
Hmm only nervous about the forno link hardcoded for alfajores. Would it work to use the `OdisUtils.Query.getServiceContext` function? And maybe have an optional env variable in the attestation service, maybe like: `const odisPubKey = OdisUtils.Query.getServiceContext(fetchEnvOrDefault('NETWORK', 'mainnet'))`? Ofc difference between that and what you suggested is that baklava would default to mainnet instead of alfajores, so if that's an issue then don't have to do that either! 

This change would require setting the `NETWORK` env var for the attestation service alfajores deployment, but that should be fairly straightforward (should require a small addition in `packages/celotool/src/lib/attestation-service.ts` in `helmParameters` (can be populated using the existing `KUBERNETES_CLUSTER_NAME` env var in the `.env.alfajores` config file that's used for deployment or define a new var specific to the AS) and then in `packages/helm-charts/attestation-service/templates/attestation.statefulset.yaml` -- can look at `celo_provider_backups_str` and how it's used to populate `CELO_PROVIDERS` for an example). Wdyt?

Reply:
I ended up adding an env variable and using it with `getServiceContext`. Let me know if that looks good!